nomos-cli/src/cmds/disseminate/mod.rs

```diff
@@ -185,7 +192,25 @@ impl ServiceCore for DisseminateService {
                         );
                         std::process::exit(1);
                     }
-                    _ => {}
+                    Ok(Ok(cert)) => {
+                        if let Some(node_addr) = node_addr {
+                            let client = Client::new();
+                            tracing::info!("Sending certificate to node...");
+                            let res = client
+                                .post(node_addr.join(NODE_CERT_PATH).unwrap())
+                                .body(wire::serialize(&cert).unwrap())
+                                .send()
+                                .await?;
+                            if !res.status().is_success() {
+                                tracing::error!(
+                                    "Failed to send certificate to node: {}",
+                                    res.status()
+                                );
+                                std::process::exit(1);
+                            }
+                            terminal_cmd_done();
+                        }
+                    }
```

Comment:
If we do not send the certificate, shouldn't we at least save it or something? 🤔

Reply:
right, probably

Reply:
Nitpick:  what if both options are activated? 🤔